### PR TITLE
Add PA state and counties to outlier exclusion list

### PR DESCRIPTION
**Pull request by smcclure17**

### CODE REVIEW

File: data/region-overrides.json [hunk 1]
```diff
@@ -807,15 +807,6 @@
...
       "context": "NY County, NY big assumed backlog on 5/9/2022"
     },
-    {
-      "include": "region",
-      "metric": "metrics.caseDensity",
-      "blocked": true,
-      "start_date": "2022-05-10",
-      "end_date": "2022-05-10",
-      "region": "PA",
-      "context": "PA staet-level cases spike on 5/11/2022"
```
Comment (mikelehen) [inline data/region-overrides.json:817]: Oh, haha. I guess we intentionally removed this as an outlier.  I somehow missed that and assumed it was outlier detection. 🤦

File: libs/datasets/outlier_detection.py [hunk 2]
```diff
@@ -14,8 +14,12 @@
 
 MultiRegionDataset = timeseries.MultiRegionDataset
 
-# Florida biweekly case reporting causes outlier detection to trigger inadvertently.
-STATES_TO_EXCLUDE_CASES_DEATHS = [RegionMask(AggregationLevel.COUNTY, states=["FL"])]
+STATES_TO_EXCLUDE_CASES_DEATHS = [
+    # FL: Biweekly case reporting causes outlier detection to trigger inadvertently.
+    # PA: Switching to weekly case reporting caused outlier detection to trigger inadvertently.
```
Comment (mikelehen) [inline libs/datasets/outlier_detection.py:19]: Per the comment above, it's possible we actually don't need to do this.  But I'm fine either way. 🤷

**Review by mikelehen — APPROVED** [review]

File: data/region-overrides.json (hunk 1 above)
Comment (smcclure17) [reply data/region-overrides.json:817]: oops 😭 looks like I added that but I have no recollection of it--sorry for not realizing that hahah 🤦

File: libs/datasets/outlier_detection.py (hunk 2 above)
Comment (smcclure17) [reply libs/datasets/outlier_detection.py:19]: Yeah, agreed this probably isn't necessary--I'll remove all the blocks and create a snapshot to see what that looks like

File: data/region-overrides.json (hunk 1 above)
Comment (mikelehen) [reply data/region-overrides.json:817]: Heh. Nah, it's 100% my bad for not noticing it when I was diagnosing the issue of low cases. All good.

**Pull request closed without merging**